src/main/java/org/csanchez/jenkins/plugins/kubernetes/PodTemplateBuilder.java

```diff
@@ -258,6 +260,8 @@ public Pod build() {
                 LOGGER.log(Level.INFO, "Computer is null for agent: {0}", slave.getNodeName());
             }
 
+            env.put("JENKINS_AGENT_WORKDIR", workingDir);
```

Comment:
Is this necessary? I thought just setting the remote FS would suffice.

Reply:
Not strictly necessary, just thought it would be nicer